docs/source/en/api/pipelines/kandinsky3.md

```diff
@@ -9,7 +9,25 @@ specific language governing permissions and limitations under the License.
 
 # Kandinsky 3
 
-TODO
+Kandinsky 3 is created by [Arkhipkin Vladimir](https://github.com/oriBetelgeuse), [Igor Pavlov](https://github.com/boomb0om), [Andrei Filatov](https://github.com/anvilarth), [Zein Shaheen](https://github.com/zeinsh).
```

Comment:
Let's add the entire list of authors here https://github.com/ai-forever/Kandinsky-3#authors

Reply:
Ahh, I missed it. @yiyixuxu Thank you for pointing out :)